### ISSUE TITLE
Add CSScomb beautifier

### ISSUE THREAD

**Issue by Glavin001** [Unibeautify/vscode#136]
_From @muuvmuuv on September 28, 2018 15:46_

Noticed that it is not yet added to the beautifiers:

```
Extension Host] Error: Beautifier not found: CSScomb
    at Unibeautify.beautify (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/node_modules/unibeautify/src/beautifier.ts:309:9)
    at EditProvider.beautifyOptions.then.beautifyOptions (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/src/EditProvider.ts:80:28)
    at <anonymous>
t.log @ console.ts:136
t._logExtensionHostMessage @ extensionHost.ts:412
(anonymous) @ extensionHost.ts:224
emitTwo @ events.js:126
emit @ events.js:214
emit @ internal/child_process.js:772
_combinedTickCallback @ internal/process/next_tick.js:141
_tickCallback @ internal/process/next_tick.js:180
log.ts:171   ERR Beautifier not found: CSScomb: Error: Beautifier not found: CSScomb
    at Unibeautify.beautify (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/node_modules/unibeautify/src/beautifier.ts:309:9)
    at EditProvider.beautifyOptions.then.beautifyOptions (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/src/EditProvider.ts:80:28)
    at <anonymous>
```
_Copied from original issue: Unibeautify/beautifier-csscomb#11_

**Comment by Glavin001** [comment]
This issue belongs in https://github.com/Unibeautify/vscode 

It is fairly trivial: https://github.com/Unibeautify/vscode/commit/c827224e35376feb6bc9181e678e2afe1aebfb7c

Would you like to make the Pull Request? 😃  Let me know if you have any questions!

**Comment by muuvmuuv** [comment]
@Glavin001  TSLint seems to be missing too

```
[Extension Host] beautifyData {fileExtension: ".ts", filePath: "/Users/marvinheilemann/Documents/Customers/Additive/Tools/origin-version-to-html/index.ts", languageName: "TypeScript", options: {…}, projectPath: "/Users/marvinheilemann/Documents/Customers/Additive/Tools/origin-version-to-html", …}
2console.ts:136 [Extension Host] Error: Beautifier not found: TSLint
    at Unibeautify.beautify (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/node_modules/unibeautify/src/beautifier.ts:309:9)
    at EditProvider.beautifyOptions.then.beautifyOptions (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/src/EditProvider.ts:80:28)
    at <anonymous>
t.log @ console.ts:136
t._logExtensionHostMessage @ extensionHost.ts:438
(anonymous) @ extensionHost.ts:246
emitTwo @ events.js:126
emit @ events.js:214
emit @ internal/child_process.js:772
_combinedTickCallback @ internal/process/next_tick.js:141
_tickCallback @ internal/process/next_tick.js:180
log.ts:171   ERR Beautifier not found: TSLint: Error: Beautifier not found: TSLint
    at Unibeautify.beautify (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/node_modules/unibeautify/src/beautifier.ts:309:9)
    at EditProvider.beautifyOptions.then.beautifyOptions (/Users/marvinheilemann/.vscode/extensions/glavin001.unibeautify-vscode-0.6.1/src/EditProvider.ts:80:28)
    at <anonymous>
```

**Comment by stevenzeck** [comment]
TSLint won't be added until the beautifier is more complete.

**Comment by muuvmuuv** [comment]
@Glavin001 I think you need to make a `vsce publish` now to upload the changes to the plugin.

**Comment by Glavin001** [comment]
@muuvmuuv could you elaborate why I need to publish?

Unfortunately, I've been busy and haven't taken a look at Unibeautify VSCode recently so I may be out of the loop with what you and @stevenzeck and working on here 😄 . Thanks!

**Comment by Glavin001** [comment]
Published to v0.7.0
